publisher-sdk-tests/src/androidTest/java/com/criteo/publisher/BidManagerFunctionalTest.java

```diff
@@ -192,6 +193,42 @@ public void prefetch_GivenAdUnits_ShouldCallCdbAndPopulateCache() throws Excepti
     assertShouldCallCdbAndPopulateCacheOnlyOnce(mappedAdUnitsChunks.get(0), slot);
   }
 
+  @Test
+  public void prefetch_GivenRateLimited_ShouldNotCallCdbAndPopulateCache() throws Exception {
```

Comment:
Do you mind adding a test showing that after the clock has been advanced, then it is possible to make again a prefetch.